application/artifacts/artifact_creation.py

```diff
@@ -0,0 +1,110 @@
+"""Abstracting Sync of Neo and Elastic"""
+import datetime
+import os
+import uuid
+
+import werkzeug
+from PIL import Image
+from flask import current_app
+
+from application.artifacts.artifact_connector import ArtifactConnector
+
+from application.artifacts.image_recognition import ImageRecognizer
+
+
+class ImageResizer:  # pylint:disable=too-few-public-methods
+    """Saves resized versions of an image to disk"""
+
+    widths = [320, 480, 640, 750, 1080]
+
+    def __init__(self, file_path):
+        self.image = Image.open(file_path)
+        self.original_file_name = file_path.split("/")[-1]
+
+    def save_sizes(self):
+        """Saves resized image for all defined widths"""
+        for width in self.widths:
+            height = self._calculate_height(width)
+            resized_image = self.image.resize((width, height), Image.ANTIALIAS)
+            resized_image.save(self._generate_file_path(width))
+            resized_image.close()
+
+    def _generate_file_path(self, width):
```

Comment:
Could add docstring here just for consistency.

Reply:
I wont add a docstring to private methods (except if it isn't clear what the are doing by method name). But I added a class comment and one to the public method.

Reply:
👍